crates/usvg-parser/src/converter.rs

```diff
@@ -18,6 +18,8 @@ use crate::{Error, Options};
 pub struct State<'a> {
     pub(crate) parent_clip_path: Option<SvgNode<'a, 'a>>,
     pub(crate) parent_markers: Vec<SvgNode<'a, 'a>>,
+    pub(crate) context_stroke: Option<Stroke>,
```

Comment:
We have to make sure it would work with nested `use` and `marker`.

Reply:
I added some more tests.